src/main/java/com/conveyal/r5/streets/EdgeTraversalTimes.java

```diff
@@ -106,4 +107,17 @@ public double getWalkTimeFactor (int edgeIndex) {
     public double getBikeTimeFactor (int edgeIndex) {
         return bikeTraversalTimes.perceivedLengthMultipliers.get(edgeIndex);
     }
+
+    /**
+     * Factory method returning a newly constructed EdgeTraversalTimes where all scaling factors are 1 and constant
+     * costs are 0. This serves as a neutral starting point for building up generalized costs in modifications,
+     * as opposed to starting from pre-existing generalized costs derived from special purpose OSM tags.
+     */
+    public static EdgeTraversalTimes unity (EdgeStore edgeStore) {
+        var times = new EdgeTraversalTimes(edgeStore);
+        times.bikeTraversalTimes.setAllUnity();
+        times.walkTraversalTimes.setAllUnity();
```

Comment:
Additionally, this section could be replaced with
```suggestion
        for (int i = 0; i < edgeStore.nEdges(); i++) {
            times.bikeTraversalTimes.setOneEdge();
            times.walkTraversalTimes.setOneEdge();
        }
```
and the new `setAllUnity` method could be removed.

Reply:
I think I just had a reflex of aiming for better encapsulation ("tell the object for each mode to set its own internal fields") but the EdgeTraversalTimes and SingleModeTraversalTimes are already essentially friend classes. If it's easier to understand without the extra method I'll remove it. I'll rename the existing method though because "setOneEdge" should be something like "addOneNeutralEdge".